modules/cpmso-yaml-provider-spec-nutanix.adoc

```diff
@@ -78,22 +78,44 @@ You must use the `Legacy` boot type in {product-title} {product-version}.
 ====
 Clusters that use {product-title} version 4.15 or later can use failure domain configurations.
 
-If the cluster is configured to use a failure domain, this parameter is configured in the failure domain.
+If the cluster uses a failure domain, configure this parameter in the failure domain.
 If you specify this value in the provider specification when using failure domains, the Control Plane Machine Set Operator ignores it.
 ====
 <4> Specifies the secret name for the cluster. Do not change this value.
 <5> Specifies the image that was used to create the disk.
 <6> Specifies the cloud provider platform type. Do not change this value.
 <7> Specifies the memory allocated for the control plane machines.
 <8> Specifies the Nutanix project that you use for your cluster. In this example, the project type is `name`, so there is a `name` stanza.
-<9> Specifies a subnet configuration. In this example, the subnet type is `uuid`, so there is a `uuid` stanza.
+<9> Specify one or more Prism Element subnet objects.
+In this example, the subnet type is `uuid`, so there is a `uuid` stanza.
+A maximum of 32 subnets for each Prism Element failure domain in the cluster is supported.
++
+[IMPORTANT]
+====
+The following known issues with configuring multiple subnets exist in {product-title} version 4.18:
+
+* Adding subnets above the existing subnet in the `subnets` stanza causes a control plane node to become stuck in the `SchedulingDisabled` state.
+As a workaround, only add subnets below the existing subnet in the `subnets` stanza.
+
+* Sometimes, after adding a subnet, the updated control plane machines appear in the Nutanix console but the {product-title} cluster is unreachable.
+There is no workaround for this issue.
+
+For more information, see link:https://issues.redhat.com/browse/OCPBUGS-50904[*OCPBUGS-50904*].
+====
++
+The CIDR IP address prefix for one of the specified subnets must contain the virtual IP addresses that the {product-title} cluster uses. 
+All subnet UUID values must be unique.
 +
 [NOTE]
 ====
 Clusters that use {product-title} version 4.15 or later can use failure domain configurations.
 
-If the cluster is configured to use a failure domain, this parameter is configured in the failure domain.
+If the cluster uses a failure domain, configure this parameter in the failure domain.
 If you specify this value in the provider specification when using failure domains, the Control Plane Machine Set Operator ignores it.
+
+There is a known issue in {product-title} version 4.18 that prevents configuring multiple subnets in a failure domain.
+There is no workaround for this issue.
```

Comment:
Note that OCPBUGS-49885 only affects day1 cluster installation when configuring more than one subnets with failure domain using the installer. For the day2 CPMS multi-subnets update issue, the issue is OCPBUGS-50904 mentioned above.

Reply:
ok - so is it fine to do this with failure domains in day-2? (aside from the other issue in 50904)

Reply:
hm, part of the issue here is that "Adding failure domains to an existing Nutanix cluster" (day-2) is in the install docs. I don't know why that happened (looking into it). But seems if this works for day-2, I can revert any of my 49885 changes since they are all for day-2

Reply:
thanks - I am pushing a change to clarify that 50904 applies to CPMS day-2 configs regardless of if they are using the `providerSpec` or failure domains to specify the subnets (here and in the rel notes)

Reply:
Thank you @jeana-redhat LGTM